### PR TITLE
- get name and age of enum value

### PR DESCRIPTION
**Pull request by AndrewQA88**
- get name and age of enum value
- get all values from enum (values)

### CODE REVIEW

File: src/test/java/exercises/Iryna.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package exercises;
+
+public enum Iryna {
+    IRA("Ira", 30), IRYNA("Iryna", 30), IRUSYA("Irusya", 30);
```
[the remark this answers is not in the document]
Comment (AndrewQA88) [reply src/test/java/exercises/Iryna.java]: Updated:
`public enum ContactInformation {
    IRA("Ira", 30),
    JHON("Jhon", 35),
    JACK("Jack", 49);`

File: src/test/java/exercises/ContactInformation.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package exercises;
+
+public enum ContactInformation {
+    IRA("Ira", 30),
+    JHON("Jhon", 35),
+    JACK("Jack", 49);
+
+    private final String name;
```
Comment (iikarak) [inline src/test/java/exercises/ContactInformation.java:8]: remove final

File: src/test/java/exercises/ContactInformation.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package exercises;
+
+public enum ContactInformation {
+    IRA("Ira", 30),
```
Comment (iikarak) [inline src/test/java/exercises/ContactInformation.java:4]: what if you know person name but don`t know age?

File: src/test/java/exercises/ContactInformation.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    JHON("Jhon", 35),
+    JACK("Jack", 49);
+
+    private final String name;
+    private final int age;
+
+    ContactInformation(String name, int age) {
+        this.name = name;
```
Comment (iikarak) [inline src/test/java/exercises/ContactInformation.java:12]: return based on name that is String enum value of ContactInformation

**Review by iikarak — CHANGES_REQUESTED** [review]

File: src/test/java/exercises/ContactInformation.java (hunk 4 above)
Comment (AndrewQA88) [reply src/test/java/exercises/ContactInformation.java:12]: done

File: src/test/java/exercises/ContactInformation.java (hunk 2 above)
Comment (AndrewQA88) [reply src/test/java/exercises/ContactInformation.java]: done

**Pull request merged**